python/lsst/ts/Dome/dome_csc.py

```diff
@@ -108,44 +126,88 @@ def __init__(
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
-    async def do_stopAz(self, data):
+    async def do_park(self, data):
         """
-        Stop AZ
+        Park
         """
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
-    async def do_stopEl(self, data):
+    async def do_setTemperature(self, data):
         """
-        Stop El
+        Set Temperature
         """
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
-    async def do_stop(self, data):
+    async def config(self, data):
         """
-        Stop
+        Internal Config command
         """
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
-    async def do_setTemperature(self, data):
+    async def fans(self, data):
         """
-        Set Temperature
+        Internal Fans command
         """
         self.assert_enabled()
         raise salobj.ExpectedError("Not implemented")
 
+    async def inflate(self, data):
+        """
+        Internal Inflate command
```

Comment:
This is not very informative. What does "Internal" mean in this context? What does "Inflate" mean?

Reply:
I added more descriptive inline comments for these "internal" functions.